mesh/ncmesh.cpp

```diff
@@ -811,6 +853,13 @@ void NCMesh::RefineElement(int elem, char ref_type)
       return;
    }
 
+   /*std::cout << "Refining element " << elem << " ("
```

Comment:
Perhaps this is no longer needed

Reply:
I would actually like to keep it for future debugging...